spiceaidocs/content/en/reference/Connectors/s3.md

```diff
@@ -0,0 +1,44 @@
+---
+type: docs
+title: "S3 Parquet Dataset"
+linkTitle: "S3 Parquet Dataset"
+description: 'S3 Parquet Dataset YAML reference'
+weight: 80
+---
+
+Parquet data in S3-compatible storage can be queried in mesh mode or accelerated locally.
+
+# `dataset`
+
+```yaml
+- from: s3://s3-bucket-name/path/to/parquet/file.parquet
+  name: s3_file
+  auth: s3
+  params:
+    endpoint: https://your.minio.host
+    region: ""
+  acceleration:
+    enabled: true
+    refresh_mode: full
+```
+
+## `params`
+
+- `endpoint`: Check `with_endpoint` in [object store](https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html)
+- `region`: Check `with_region` in [object store](https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html)
+
+## `acceleration`
```

Comment:
Acceleration is not specific to S3, we should remove this section (or reference general acceleration doc).